src/python/WMCore/ReqMgr/DataStructs/RequestStatus.py

```diff
@@ -124,6 +124,12 @@ def get_modifiable_properties(status = None):
     TODO: Currently gets the result from hardcoded list. change to get from configuration or db
     """
     if status:
-        return ALLOWED_ACTIONS_FOR_STATUS.get(status, 'all')
+        return ALLOWED_ACTIONS_FOR_STATUS.get(status, 'all_attributes')
     else:
         return ALLOWED_ACTIONS_FOR_STATUS
+
+def get_protected_properties():
```

Comment:
Sorry that I missed this PR. But these are not the only protected properties that we have for a request, though I don't know if we really care.